src/Core/SettingsChangesHistory.h

```diff
@@ -80,8 +80,10 @@ namespace SettingsChangesHistory
 /// It's used to implement `compatibility` setting (see https://github.com/ClickHouse/ClickHouse/issues/35972)
 static std::map<ClickHouseVersion, SettingsChangesHistory::SettingsChanges> settings_changes_history =
 {
-    {"23.4", {{"allow_suspicious_indices", true, false, "If true, index can defined with identical expressions"}}},
-    {"23.4", {{"connect_timeout_with_failover_ms", 50, 1000, "Increase default connect timeout because of async connect"},
+    {"23.5", {{"input_format_parquet_preserve_order", true, false, "Allow Parquet reade to reorder rows for better parallelism."},
```

Comment:
`reade` - typo

Reply:
Oops. I'll piggy-back the fix to some unrelated PR. (Unless it's important enough for its own PR, or we have some different process for this, or sth.)